src/platform/lumin-runtime/native-linking.js

```diff
@@ -0,0 +1,26 @@
+import { DispatchResult, TryOpenApplication } from 'lumin';
+
+class NativeLinking {
+  canOpenURL(url) {
+    return new Promise((resolve, reject) => {
```

Comment:
Why is this a promise?

Reply:
Because React-Native Linking module also returns a promise. React Native requires these 2 functions to be async so we need the same for Lumin, don't we? We can use either promises or async/await.